### PR TITLE
fix typo in footer + imrpove hero style

### PR DESCRIPTION
**Pull request by rdubigny**

### CODE REVIEW

**Comment by rdubigny** [comment]
[Retravailler particulier.api.gouv.fr pour qu'il parle au maire](https://trello.com/c/fBqH6LSM/198-retravailler-particulierapigouvfr-pour-quil-parle-au-maire)

File: FAQ.html [hunk 1]
```diff
@@ -219,7 +219,7 @@ <h2 class="section-heading">
             <li><a href="https://particulier.api.gouv.fr/FAQ.html">FAQ</a></li>
             <li><a href="https://status.particulier.api.gouv.fr">Status des API</a></li>
             <li><a href="https://api.gouv.fr/apropos#mentions-légales">Mentions légales</a></li>
-            <li><a href="mailto:contact@particulier.api.gouv.fr?subject=Contact%20via%20particulier.api.gouv.fr">Contactez nous</a></li>
+            <li><a href="mailto:contact@particulier.api.gouv.fr?subject=Contact%20via%20particulier.api.gouv.fr">Contactez-nous</a></li>
```
Comment (Anna-Livia) [inline FAQ.html:222]: 🙌

File: index.html [hunk 2]
```diff
@@ -120,13 +120,13 @@ <h1>
     </div>
 </div>
 
-<section class="section-color">
+<section class="">
```
Comment (Anna-Livia) [inline index.html:123]: Why not delete ` class=""` and replace with a meaningfull `id` ?

File: css/main.css [hunk 3]
```diff
@@ -22,10 +21,6 @@
     justify-content: center;
 }
 
-.hero .illustration {
```
Comment (Anna-Livia) [inline css/main.css:25]: Removing this makes the illustration disapear on my browser (Firefox V62)
capture d ecran 2018-09-24 a 14 02 31

**Review by Anna-Livia — CHANGES_REQUESTED** [review]

**Comment by Anna-Livia** [comment]
The look & feel of the new design appears very differently on my browsers : 
Here is Chrome Before/after : 
chrome-before

capture d ecran 2018-09-24 a 14 05 15

Here is Firefox Before/after:

ff-before
capture d ecran 2018-09-24 a 14 02 31

Here is Safari (Before/my favorite after
safari-before
):

capture d ecran 2018-09-24 a 14 07 01

File: index.html (hunk 2 above)
Comment (rdubigny) [reply index.html:123]: this is a mix of 2 distinct points: 

- there is no class to define section-grey (as section-white do) so I kept it here to make it more clear that no class means grey
- we should add id on every section but I prefer to do it in another PR as we are not sure of the content yet

Comment (Anna-Livia) [reply index.html:123]: I think `<section class="default_to_grey">` could be even more explicit. WDYT?

Agreed on point #2

**Review by Anna-Livia — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (rdubigny) [reply index.html:123]: We discussed this with Thimy, to include it in the template stylesheet. But there is too much work for limited gain.

**Pull request merged**